### PR TITLE
Ignore ssl errors on HttpClient

### PR DESCRIPTION
**Pull request by alesebi91**
Add option (default 'false') to ignore HttpClient SSL certificate errors when contacting nuget.org

### CODE REVIEW

**Comment by tomchavakis** [comment]
Hi @alesebi91,

Thank you for the contribution.
Is it possible to add a Unit Test for the IgnoreSslCertificateErrors, please?
Moreover, I am curious why you need to disable the SSL Verification and the only reason that I can think of is that you are using a local NuGet server without SSL in your stack. Is that correct?

File: src/Methods.cs [hunk 1]
```diff
@@ -91,61 +96,66 @@ public async Task<PackageList> GetNugetInformationAsync(string project, IEnumera
                             continue;
                         }
 
-                        // Search nuspec in local cache
+                        // Search nuspec in local cache (Fix for linux distro)
                         string userDir = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile);
-                        var nuspecPath = Path.Combine(userDir, ".nuget", "packages", packageWithVersion.Name, version, packageWithVersion.Name + ".nuspec");
+
+                        //Linux: package file name is lowercase
+                        var packageNameTmp = IsLinux() ? packageWithVersion.Name?.ToLowerInvariant() : packageWithVersion.Name;
```
Comment (tomchavakis) [inline src/Methods.cs:103]: I am not sure if there is any difference between Linux and Windows for NuGet. I've checked both Ubuntu and Windows packages folder (.nuget\packages) and all package names are the same. Is there any example?

**Comment by alesebi91** [comment]
Hi @tomchavakis,
yes, I will write Unit Test shortly.

I need to disable SSL verification because of issues when you are behind corporate proxy that changes root CA for nuget.org source (CI/CD inside containers) or because you have private nuget sources (or nuget proxy) in the CI/CD or development stack.

**Review by tomchavakis — APPROVED** [review]

**Comment by tomchavakis** [comment]
Hi @alesebi91,

Thank you for the response and the Unit Test.
Is it ok to merge it? Do you want to add anything else?

**Comment by alesebi91** [comment]
Hi @tomchavakis,
if you want you can merge pull #72 and #71 to create a new library version.

Thank you!

**Pull request merged**